hypertrace-trace-enricher/hypertrace-trace-enricher-impl/src/main/java/org/hypertrace/traceenricher/enrichment/enrichers/resolver/backend/HttpBackendResolver.java

```diff
@@ -51,7 +58,13 @@
           event, entityBuilder, HttpSemanticConventionUtils.getAttributeKeysForHttpMethod());
       setAttributeForFirstExistingKey(
           event, entityBuilder, HttpSemanticConventionUtils.getAttributeKeysForHttpRequestMethod());
-      return Optional.of(entityBuilder.build());
+
+      Map<String, AttributeValue> enrichedAttributes = new HashMap<>();
+      String httpOperation = httpRequest.map(Request::getMethod).orElse(null);
```

Comment:
This is so confusing! 

@kotharironak @rish691 @JBAhire What did we decide on first class fields on the Event? Are we going with it or getting rid of them?

The last status I know was we were getting rid of the first class fields

Reply:
Yes, but in this method, it is used. So, I was also wondering, but as it is used in this resolver, it's better to refactor complete things than just one place.

Reply:
> The last status I know was we were getting rid of the first class fields

We want to get rid of it and rely on attributes from attribute_map, but the extent of its usage across makes it difficult to remove completely.

Reply:
Right. How do we start getting rid of it though? Using something like a new utils, RawAttributeUtils? Just like EnrichedSpanUtils